### ISSUE TITLE
bzg

### ISSUE THREAD

**Issue by mattbk** [gratipay/team-review#42]
https://gratipay.com/bzg/

@bzg, your team application is being reviewed here.

**Comment by bzg** [comment]
What does it mean?  I get an "401 unauthorized" error on that page.

**Comment by chadwhitacre** [comment]
@bzg Sorry for the confusion. If you sign in at upper right you should be able to see the page.

**Comment by bzg** [comment]
I see.  So the review is still pending?  Thanks.

**Comment by chadwhitacre** [comment]
Yup, still pending. :-)

It looks like you made a Team that represents you personally. A Team should represent a product or service, not an individual person. It looks like your main project is [Org-mode](http://orgmode.org/). Does it make sense to apply for a Team for Org-mode?

**Comment by bzg** [comment]
Yes, it makes sense.  Should I do this modification myself or can you do it?

**Comment by chadwhitacre** [comment]
> Yes, it makes sense.

Cool. :-)

> Should I do this modification myself or can you do it?

I've rejected your bzg Team application. You can go ahead and [apply for a new one](https://gratipay.com/new) for Org-mode.

**Comment by bzg** [comment]
Done, thanks.

**Comment by chadwhitacre** [comment]
@bzg Would you like to reopen this ticket in light of the decision to allow personally branded Teams (#82)?

**Comment by bzg** [comment]
@whit537 Yes.  Actually, the Org-mode team makes a lot of sense, and I plan to advertize it on the org-mode mailing list soon, so that others can enjoy gittip donations through it.  But it also makes sense for me to have a personally branded Team/Project.

Let me know what action I need to take on top of this reply.

**Comment by chadwhitacre** [comment]
@bzg Okay, reopening and I'll aim to get this squared away next week. I'll let you know if we need

**Comment by chadwhitacre** [comment]
... anything else from you. :-)

**Comment by bzg** [comment]
Great - thanks!

**Comment by chadwhitacre** [comment]
@bzg Can we talk links? Onboarding and to-do both point to https://bzg.fr/about.en.html, which is 404 (I'm not sure but you probably created the Team before we even added those URLs at all). Do you have links we could use for those? Maybe this one for to-do?

https://github.com/issues?utf8=%E2%9C%93&q=is%3Aopen+user%3Abzg

Also, I grabbed a picture from your website. Let me know if you'd like to use something different.

**Comment by bzg** [comment]
Sorry for the late reply.

Here is a personal todo list: http://bzg.fr/todo.html

And an updated about page: http://bzg.fr/author/bzg-en/

Let me know if you need anything else!  And thanks again.

**Comment by chadwhitacre** [comment]
@bzg Hmm ... so, I think we're still crossing wires. Every Team on Gratipay is an open organization, even if it's named after an individual person—"open" meaning that anyone at any time can start working for your organization without having to explicitly coordinate with you first. In other words, you should think of `bzg` on Gratipay as "a company named after you," not simply "you." I've been unpacking this further at  https://github.com/gratipay/gratipay.com/issues/3433#issuecomment-159934765.

Does that make any sense?

**Comment by bzg** [comment]
> anyone at any time can start working for your organization without having to explicitly coordinate with you first

Yes, that's the case.  Most projects listed on http://bzg.fr/todo.html are collective projects where people already contribute by themselves.

I'm not sure I groke the spirit of the change I should make... can you give an example?

**Comment by chadwhitacre** [comment]
> I'm not sure I groke the spirit of the change I should make... can you give an example?

@bzg Sorry ... I was looking specifically at the about page: http://bzg.fr/author/bzg-en/. For the onboarding URL we're looking for something that tells potential collaborators how to get involved in your work. The About page as written seems more addressed to potential clients.

I've updated the todo URL and reset the Team to "Under Review."

**Comment by bzg** [comment]
Yes, http://bzg.fr/author/bzg-en/ is more oriented toward getting new clients, while http://bzg.fr/todo.html tells more about the free software projects I'm now involved in.  I'll try to enhance this page progressively.

**Comment by mattbk** [comment]
@bzg, I'm happy to say that we have the [new ToS](https://gratipay.com/about/policies/terms-of-service) in place, which means I can approve this project (teams are now projects).

I made one small change to your profile based on the new ToS, which focuses on matching donations to the product or service you deliver. I replaced the homepage URL with your old todo URL of https://bzg.fr/todo.html. The todo and onboarding URLs have been removed. Let me know if you think this presents a problem.
